dataflow/gpu-workers/tensorflow-landsat/main.py

```diff
@@ -276,39 +276,42 @@ def run(
     max_value = vis_params["max"]
     gamma = vis_params["gamma"]
 
-    options = PipelineOptions(beam_args, save_main_session=True)
-    with beam.Pipeline(options=options) as pipeline:
-        # Optionally, validate that the workers are using GPUs.
-        gpu_check = (
-            pipeline
-            | beam.Create([None])
-            | "Check GPU availability" >> beam.Map(check_gpus, gpus_optional)
+    beam_options = PipelineOptions(beam_args, save_main_session=True)
+
+    # We currently cannot use the `with` statement to run without waiting.
```

Comment:
I think users may not understand this comment, let's remove it. Re `with` behavior, that may be WAI - for example when I access files in a `with` statement, resources are cleaned up after we exit from the context, so it makes sense to me that the `with` statement waits for pipeline completion.

Reply:
Alright, I'm removing this comment for now.